keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -187,9 +176,6 @@ able to manually apply the `triage/accepted` label.)
 
 **Considerations:**
 
-- `triage/accepted` should be considered a placeholder name for the
-  label in this provisional state, depending on what we decide is the most
-  appropriate name.
 - There will be an expectation that only SIG members designated to triage
   issues are applying triage labels. We're considering limiting application of
   this label to the [`milestone-maintainers` GitHub team](https://github.com/orgs/kubernetes/teams/milestone-maintainers).
```

Comment:
The milestone maintainers is a highly selective group of people and currently involves the release team + SIG leads. I think we should open this up to all org members.

Reply:
I think the idea behind this thought was to ensure its actually being triaged by the SIGs. - We could start with it open to everyone and if misuse is common enough fall back to limiting it to milestone maintainers or another group that has the approval to triage.